optimize/backend/pom.xml

```diff
@@ -187,11 +215,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+    -->
+    <!-- unused/declared
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-actuator</artifactId>
       <version>${spring.boot.version}</version>
     </dependency>
+    -->
```

Comment:
I would be surprised if this could be removed, I think it is used to expose our Metrics. However, the metrics IT is currently in a state of skipped. Would be nice to reenable and get green again

Reply:
Maybe we could leave it commented out, for the moment, instead of deleting it and with a TODO?

Reply:
Or we just test metrics on this branch before merging. One way to do this would be reenabling the MetricsITs (might require finding out what the initial problem with it was anyway)